platform-services.md

```diff
@@ -0,0 +1,215 @@
+platform-specific native APIs, or APIs from third-party vendors. Flutter offers several ways
+for you to integrate the functionality of these of these platform services to extend your Flutter
+app.
+
+### First-party services for Flutter {#first-party-services-for-flutter}
+
+For frequently used services, the Flutter SDK provides APIs to access first-party services that
+are supported across platforms. You can call these APIs directly in your Dart code. These
+first-party services are provided through the `SKY_SERVICES` library. The following list
+summarizes these first-party services.
+
++ [`activity_mojom`](http://docs.flutter.io/flutter/activity_mojom/activity_mojom-library.html)
++ [`app_messages_mojom`](http://docs.flutter.io/flutter/app_messages_mojom/app_messages_mojom-library.html)
++ [`editing_mojom`](http://docs.flutter.io/flutter/editing_mojom/editing_mojom-library.html)
++ [`haptic_feedback_mojom`](http://docs.flutter.io/flutter/haptic_feedback_mojom/haptic_feedback_mojom-library.html)
++ [`input_event_mojom`](http://docs.flutter.io/flutter/input_event_mojom/input_event_mojom-library.html)
++ [`media_mojom`](http://docs.flutter.io/flutter/media_mojom/media_mojom-library.html)
++ [`path_provider_mojom`](http://docs.flutter.io/flutter/path_provider_mojom/path_provider_mojom-library.html)
++ [`pointer_mojom`](http://docs.flutter.io/flutter/pointer_mojom/pointer_mojom-library.html)
++ [`raw_keyboard_mojom`](http://docs.flutter.io/flutter/raw_keyboard_mojom/raw_keyboard_mojom-library.html)
++ [`semantics_mojom`](http://docs.flutter.io/flutter/semantics_mojom/semantics_mojom-library.html)
++ [`system_chrome_mojom`](http://docs.flutter.io/flutter/system_chrome_mojom/system_chrome_mojom-library.html)
++ [`system_sound_mojom`](http://docs.flutter.io/flutter/system_sound_mojom/system_sound_mojom-library.html)
+
+### Platform and Third-party Services for Flutter {#platform-and-third-party-services-for-flutter}
+
+To access APIs that are native to the iOS or Android mobile platform, Flutter provides the
+`FlutterView` interface. The `FlutterView` interface allows you to embed a bi-directional
+asynchronous messaging component in your platform code to send and receive native service calls
+in a transparent way. This flexible approach lets you create cross-platform mobile apps that can
+connect to platform-specific and third-party services on iOS and Android devices, while sharing a
+common user interface (UI) codebase in Dart.
+
+You can build your own integrations with platform-specific services and third-party SDKs. Ideally,
+the services that you invoke using the `FlutterView` interface should return non-graphical,
+data-only information. Examples include:
+
+*   Geolocation
+*   Contacts
+*   File system
+*   Network information
+*   Sensors
+
+You can also use the `FlutterView` interface to invoke APIs from third-party SDKs that run on iOS
+and Android devices.
```

Comment:
In all these places that you refer to `FlutterView`, I think it would be more accurate to refer to HostMessages: http://docs.flutter.io/flutter/services/HostMessages-class.html  That's the interface that you actually use to call out to the platform-specific part of your applciation.  `FlutterView` is the interface that goes the other way, from the platform-specific part of your application to the cross-platform UI.